docs/streaming-custom-receivers.md

```diff
@@ -257,9 +257,9 @@ The following table summarizes the characteristics of both types of receivers
 
 ## Implementing and Using a Custom Actor-based Receiver
 
-Custom [Akka Actors](http://doc.akka.io/docs/akka/2.2.4/scala/actors.html) can also be used to
+Custom [Akka Actors](http://doc.akka.io/docs/akka/current/scala/actors.html) can also be used to
```

Comment:
It's about actors which are the fundamental concept of Akka so it's of less worry (like RDDs in Spark), but have changed it to the Akka version Spark uses, i.e. `2.3.11`.